src/bp/ui-admin/src/App/Layout.tsx

```diff
@@ -23,15 +25,45 @@ interface Props {
 }
 
 const App: FC<Props> = props => {
+  const [tokenInterval, setTokenInterval] = useState()
+
   useEffect(() => {
     props.fetchLicensing()
     props.fetchProfile()
 
     if (!props.version) {
       props.fetchCurrentVersion()
     }
+
+    setTokenInterval(
+      setInterval(async () => {
+        await validateToken()
+      }, REFRESH_INTERVAL)
+    )
   }, [])
 
+  const validateToken = async () => {
```

Comment:
what about naming this `tryRefreshToken` or `refreshTokenIfNeeded` ? I think this it would be cleaner validateToken since it does more than validating. What do you think ?

Reply:
Yup, i like that